extensions/amp-mathml/1.0/test/validator-amp-mathml.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
I see there's not changes to the [validator rules](https://github.com/ampproject/amphtml/blob/main/extensions/amp-mathml/validator-amp-mathml.protoascii), so this test may not be necessary. Usually changes to these files should go together, including the accompanying `.out` that results from running `amp validation --update_tests`. Feel free to add all three file changes to this PR or do them in a separate PR, but they should go together.

Reply:
FYI this `validator-amp-mathml.html` can be copied from the 0.1 folder which already includes all the attr test cases. Since the API is not changing between versions, all you'd have to do is copy and update the import script from 0.1 to 1.0.

Reply:
didn't realize this got created when scaffolding - copied over the files from 0.1 and changed the `0.1`s -> `1.0`s